e2e/interchaintestv8/types/rust_fixtures.go

```diff
@@ -0,0 +1,81 @@
+package types
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/stretchr/testify/suite"
+
+	"github.com/srdtrk/solidity-ibc-eureka/e2e/v8/testvalues"
+	ethereumtypes "github.com/srdtrk/solidity-ibc-eureka/e2e/v8/types/ethereum"
+)
+
+type RustFixtureGenerator struct {
+	shouldGenerateFixture bool
```

Comment:
It just seems that this is not very useful since all this does is disable the fixture generator. Is it not possible to just not create the fixture generator if this is false.

Reply:
This used to be the case. But the way we do it now, it would indeed be possible.
I can do it in the next PR if that works for you?

Reply:
Would be nice whenever you get the time